atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransactionManager.java

```diff
@@ -85,7 +85,7 @@ public void registerClosingCallback(Runnable closingCallback) {
      * use the delegate instead.
      */
     protected SerializableTransactionManager() {
-        this(null, null, null, null, null, null, null, null, null, 1);
+        this(null, null, null, null, null, null, null, null, null, 1, AtlasDbConstants.DEFAULT_TIMESTAMP_CACHE_SIZE);
     }
 
     public static SerializableTransactionManager create(KeyValueService keyValueService,
```

Comment:
I like not bloating with additional constructors, but again this is a minor dev break that we should mention.